Sources/Plasma/PubUtilLib/plAudio/plEAXListenerMod.h

```diff
@@ -51,15 +52,13 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "pnModifier/plSingleModifier.h"
 
+#include <efx.h>
+#include <efx-presets.h>
```

Comment:
It would be nice to avoid including this library header in our headers. Are there any forward declarations we can use?